lib/cocoapods-packager/spec_builder.rb

```diff
@@ -15,11 +15,17 @@ def framework_path
       end
     end
 
+    def library_path
+      'lib' + @spec.name + '.a'
+    end
+
     def spec_platform(platform)
       fwk_base = platform.name.to_s + '/' + framework_path
+      library_base = platform.name.to_s + '/' + library_path
       spec = <<RB
   s.#{platform.name}.deployment_target    = '#{platform.deployment_target}'
   s.#{platform.name}.vendored_framework   = '#{fwk_base}'
+  s.#{platform.name}.vendored_library     = '#{library_base}'
```

Comment:
shouldnt we only have the framework when building frameworks, and the static library when building static libraries? rather than always having both?

Reply:
`vendored_framework` would be empty when building static libraries and vice versa.

Reply:
no it wouldnt, since we're always setting it to a string there

Reply:
Oh, I mean, the string is there, but the packaged pod wouldn't include the file, and CocoaPods seems to be OK with that.